r-pkg/uptasticsearch/NEWS.md

```diff
@@ -0,0 +1,19 @@
+# uptasticsearch 0.1.0
+
+## First release!
```

Comment:
Maybe remove this line and the description for the package below. Put the other sub sections inside a section named like `Features`.